projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -8,25 +8,45 @@ import { sortBy } from './go-table-utils';
   templateUrl: './go-table.component.html',
   styleUrls: ['./go-table.component.scss']
 })
-export class GoTableComponent implements OnChanges {
+export class GoTableComponent implements OnInit, OnChanges {
 
   @Input() tableConfig: GoTableConfig;
 
   @ContentChildren(GoTableColumnComponent) columns: QueryList<GoTableColumnComponent>;
 
   localTableConfig: GoTableConfig;
+  showTable: boolean = false;
+
+  ngOnInit() {
+    if (!this.tableConfig) {
+      throw new Error("GoTableComponent: tableConfig is a required Input");
+    } else {
+      this.renderTable();
+    }
+  }
 
   ngOnChanges() {
     this.renderTable();
   }
 
   renderTable() {
-    if (this.tableConfig!.tableData) {
+    if (this.tableConfig) {
       this.localTableConfig = JSON.parse(JSON.stringify(this.tableConfig));
+      this.showTable = true;
       this.handleSort();
+    } else {
+      this.showTable = false;
```

Comment:
Could we write this a little more concisely?

```javascript
if (this.tableConfig) {
  this.localTableConfig = JSON.parse(JSON.stringify(this.tableConfig));
  this.handleSort();
}

this.showTable = Boolean(this.tableConfig);
```

Reply:
This makes sense, updated.